R/plot_scores_scorecard.R

```diff
@@ -133,9 +133,9 @@ plot_scores_pyramide <- function() {
   p
 }
 
-annotation_df <- function() {
+annotation_df <- function(facet) {
   df <- tibble::tibble(
-    asset_class = c("bonds", "equity"),
+    asset_class = facet,
     text = c(
       "",
       "Science-based\nconsensus on\nneeded global\nambition, i.e.\nScenario [XY]"
```

Comment:
is the comment about the scenario also valid for the real estate sector?

Reply:
I don't think it is, but I didn't want to try removing it on my own 😬